packages/react-admin/src/sideEffect/saga/crudResponse.js

```diff
@@ -17,6 +17,12 @@ import {
 import { showNotification } from '../../actions/notificationActions';
 import resolveRedirectTo from '../../util/resolveRedirectTo';
 
+function* handleNotification(requestPayload, ...notificationArgs) {
+    yield requestPayload.skipNotification
```

Comment:
why not yielding the `put` only if condition matches and do nothing otherwise ?

Reply:
Yeah, I was asking myself that question, mainly kept it because of copy paste from original.